src/index.html

```diff
@@ -5,8 +5,328 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&ampdisplay=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="logo this site">
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#" class="nav_link is-active">
+              apple
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav_link">
+              samsung
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav_link">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              href="#"
+              class="nav_link"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav_link">
+              гаджеты
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav_link">
+              планшеты
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav_link">
+              фото
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a href="#" class="nav_link">
+              видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="products">
+      <div class="products__card card" data-qa="card">
+        <div class="card__img"></div>
+        <h2 class="card__name">
+          Моноблок APPLE A1419 iMac
+          27" Retina 5K (MNED2UA/A)
+        </h2>
+        <span class="card__code">
+          Код товара: 195434
+        </span>
+        <div class="card__rating">
+          <div class="card__stars">
+            <div class="card__star"></div>
```

Comment:
Тут має бути цей клас і при цьому зірочки мають змінюватися якщо змінити тут клас не міняючи css
```suggestion
        <div class="card__rating">
          <div class="card__stars stars--4">
            <div class="card__star"></div>
```